spec.emu

```diff
@@ -5,7 +5,7 @@
 <script src="./spec.js"></script>
 <pre class="metadata">
 title: Proposal Title Goes Here
-stage: -1
+stage: 0
```

Comment:
```suggestion
stage: -1
```
it's not stage 0 until it's presented or on an agenda; i'd prefer that to be an explicit action.